scripts/create_auth_jks_password.sh

```diff
@@ -46,20 +46,27 @@ fi
 
 # Generate random password for jks keystore
 generate_keystore_password(){
-  echo "Generating random password for jks keystore used in authserver..."
-  # Generate random password
-  new_keystore_passwd=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9@#+' | fold -w 12 | head -n 1)
-
+  echo "Generating password for jks keystore used in authserver..."
+  # Get password from cortx-utils
+  new_keystore_passwd=$(s3cipher --use_base64 --key_len  12  --const_key  \"openldap\")
```

Comment:
check if it  return empty value and cmd return code success

Reply:
Done